libpod/boltdb_state_internal.go

```diff
@@ -146,10 +146,11 @@ func validateDBAgainstConfig(bucket *bolt.Bucket, fieldName, runtimeValue string
 				return nil
 			}
 
-			// If DB value is the empty string, check that the
-			// runtime value is the default
-			if string(keyBytes) == "" && defaultValue != "" &&
-				runtimeValue == defaultValue {
+			// If DB value is the empty string, we ought to be fine.
+			// (Though I'm honestly not sure how we got this far
+			// with an empty-string DB config)
+			if string(keyBytes) == "" {
```

Comment:
Should we update the data with the correct value?

Reply:
The problem is that we don't really know the correct value here - the DB is supposed to know, and it doesn't, so we can only hope that we're using the right storage driver.

Reply:
I think I've figured out why we're entering "" in the first place though. Going to fix that next.